apiserver/billing/odoostorage/odoo/odoo8/odoo8.go

```diff
@@ -317,15 +327,22 @@ func mapPartnersToBillingEntity(company model.Partner, accounting model.Partner)
 			},
 			LanguagePreference: "",
 		},
+		Status: status,
 	}
 }
 
-func mapBillingEntityToPartners(be billingv1.BillingEntity, countryIDs map[string]int) (company model.Partner, accounting model.Partner, err error) {
+func mapBillingEntityToPartners(ctx context.Context, be billingv1.BillingEntity, countryIDs map[string]int) (company model.Partner, accounting model.Partner, err error) {
```

Comment:
You don't need the context here, as far as I can see.

```suggestion
func mapBillingEntityToPartners(be billingv1.BillingEntity, countryIDs map[string]int) (company model.Partner, accounting model.Partner, err error) {
```

Reply:
True, but I decided to add it to both mapping functions for consistency's sake. Would you rather have it removed?

Reply:
I'd rather remove it, so that it's obvious at the call site what data the function actually accesses.

Regarding consistency: another other option would be to change `mapPartnersToBillingEntity()` to return an error and logging the error at the call site. However, we probably want to keep the current semantics of continuing with an empty status if we can't unmarshal the data from Odoo, so we can't just early return an error if unmarshalling the raw status fails.